services/order.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+from django.db import transaction
+from django.db.models import Q, QuerySet
+
+from db.models import Order, Ticket
+from services.user import find_users
+from services.movie_session import get_movie_session_by_id
+
+
+def create_order(
+    tickets: list[dict],
+    username: str,
+    date: datetime = None
+) -> None:
+    user = find_users(username=username).first()
+    if not user:
+        raise ValueError("User does not exist")
+    if len(tickets) < 1:
+        raise ValueError("Unable to create an order without any tickets")
```

Comment:
Remove these checks. You can just try to create an Order in transaction, It will give you an error anyway in such cases. What do you think?